### PR TITLE
Add csrf-token meta tag to the page

### LINKED ISSUE #92: Add CSRF token as a meta tag
See: https://laravel.com/docs/5.4/csrf
See: https://github.com/laravel/laravel/blob/master/resources/assets/js/bootstrap.js#L32

Adding CSRF token as a meta tag to the page makes it easy to set up and send AJAX custom requests. 

It can be read like this to add to request headers: `$('meta[name="csrf-token"]').attr('content');`

### PR DESCRIPTION
**Pull request by ercanozkaya**
Closes #92

### CODE REVIEW

**Comment by ercanozkaya** [comment]
@johanjanssens can you review please?

File: code/libraries/joomlatools/library/template/filter/form.php [hunk 1]
```diff
@@ -82,6 +83,23 @@ public function filter(&$text)
     }
 
     /**
+     * Adds the CSRF token to a meta tag to be used in JavaScript
+     *
+     * @param string $text Template text
+     * @return $this
+     */
+    protected function _addMetaTag(&$text)
```
Comment (johanjanssens) [inline code/libraries/joomlatools/library/template/filter/form.php]: @ercanozkaya Would call it add 'Metatag' and use lower case. For rest it looks great, is this a pseudo standard approach? Or a custom one, is there some convention on the name of the tag?

Comment (ercanozkaya) [inline code/libraries/joomlatools/library/template/filter/form.php]: @johanjanssens More or less a pseudo-standard approach indeed. Naming convention is shared between Ruby on Rails, Laravel, and a few other smaller projects.

Comment (johanjanssens) [inline code/libraries/joomlatools/library/template/filter/form.php]: Excellent. Can you make sure to also cross add this to Kodekit please.

**Pull request closed without merging**